Palace/Settings/TPPSettingsSplitViewController+OE.swift

```diff
@@ -14,14 +14,15 @@ import Foundation
 // only meant for Open eBooks.
 // - See: https://github.com/NYPL-Simplified/Simplified-iOS/pull/1070
 extension TPPSettingsSplitViewController {
-
+  typealias DisplayStrings = DisplayStrings.TPPSettingsSplitViewController
```

Comment:
Can we make type alias to `DisplayStrings` itself?

Reply:
Never mind, this file is not in any of our build targets. Created a ticket to remove unused files.

Reply:
Nice catch, missed this in the cleanup